playground/src/props.js

```diff
@@ -33,7 +34,7 @@ export default defineComponent({
       on(n0, 'click', () => handleClick)
       renderEffect(() => setText(n0, count.value))
       /** @type {any} */
-      const n1 = createComponent(child, [
+      const n1 = createComponent(child, getCurrentInstance(), [
```

Comment:
I don't think it's a good idea, to pass parent instance as an argument. We should handle the logic inside of `createComponent`.